src/index.html

```diff
@@ -5,8 +5,451 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#" class="logo__link">
+          <img src="./images/logo.png" alt="logo-img">
+        </a>
+      </div>
+    
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-value">
+            <a 
+              class="is-active" 
+              href="#Apple"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link" 
+              href="#Samsung"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link" 
+              href="#Smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              data-qa="nav-hover"
+              class="nav-link" 
+              href="#L&c"
+            >
+            laptops & computers
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link"
+              href="#Gadgets"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link" 
+              href="#Tablets"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link" 
+              href="#Photo"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link" 
+              href="#Video"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    
+    <main class="main">
+      <div class="cards-container">
+        <div class="card-container" data-qa="card">
+          <div class="product-card">
+            <div class="product-image">
+              <img
+                class="product-image__image"
```

Comment:
```suggestion
          <div class="card">
            <div class="card__image-container">
              <img
                class="card__image"
```
Please check your BEM structure
It is not correct now
https://mate-academy.github.io/style-guides/htmlcss.html#oocss-and-bem